Conceptpower+Spring/src/integration-test/resources/output/conceptSearchNoSearchParameters.json

```diff
@@ -0,0 +1 @@
+[{"message":"No valid search parameters present."}]
```

Comment:
Yes, there could be multiple error messages. Say for example if we are having an invalid pos and an invalid operator we return both the error message to user. So that's the reason I had it as an array of error messages.